testapps/testapp/src/main/res/raw/msal_config_msa_only.json

```diff
@@ -0,0 +1,13 @@
+{
+  "client_id" : "9668f2bd-6103-4292-9024-84fa2d1b6fb2",
+  "authorization_user_agent" : "DEFAULT",
+  "redirect_uri" : "msauth://com.msft.identity.client.sample.local/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
+  "authorities" : [
+    {
+      "type": "AAD",
```

Comment:
this is confusing lol. AAD but MSA only.

Reply:
I couldn't think of what to call it. lol

It is indeed technically AAD + MSA but we restrict authority to MSA only. Which could then be overridden by the developer in AT parameters. Let me know if you think of a better name for this config file

Reply:
I don't think it's a big deal :) (hence I signed off).